src/resources/2017/Robortender.ts

```diff
@@ -0,0 +1,55 @@
+import { haveFamiliar, Item, Monster, Phylum } from "kolmafia";
+import { get } from "../../property";
+import { $familiar, $item, $phylum } from "../../template-string";
+
+/**
+ * The Robortender itself
+ */
+export const familiar = $familiar`Robortender`;
+
+/**
+ * @returns Whether you have the Robortender in your terrarium/available
+ */
+export function have(): boolean {
+  return haveFamiliar(familiar);
+}
+
+const phylumDrops = new Map<Phylum, Item>([
+  [$phylum`Bug`, $item`pickled grasshopper`],
+  [$phylum`Constellation`, $item`bottle of anís`],
```

Comment:
i would recommend making anís into the item number reference rather than the item name. when log parsing there are consistently weird parsing errors when going cross-platform with special characters and i have a feeling that could (though is not guaranteed to) cause some heartburn here too.

Reply:
I _think_ this has been fixed [as of recently](https://kolmafia.us/threads/java-bug-on-some-versions-of-java-weird-things-happen-to-strings-with-special-characters.27531/), but I'll investigate further